src/features/defi/providers/osmosis/components/OsmosisManager/Lp/Deposit/components/Confirm.tsx

```diff
@@ -191,17 +206,19 @@ export const Confirm: React.FC<ConfirmProps> = ({ onNext, accountId }) => {
         {
           opportunity: osmosisOpportunity,
           fiatAmounts: [
-            state.deposit.underlyingAsset0.fiatAmount,
-            state.deposit.underlyingAsset1.fiatAmount,
+            underlyingAssetBalances[underlyingAsset0.assetId].fiatAmount,
+            underlyingAssetBalances[underlyingAsset1.assetId].fiatAmount,
           ],
           cryptoAmounts: [
             {
               assetId: underlyingAsset0.assetId,
-              amountCryptoHuman: state.deposit.underlyingAsset0.amountCryptoHuman,
+              amountCryptoHuman:
```

Comment:
If we're calling this human it should be a human value e.g stripped digits, let's call it precision for consistency?

Reply:
Disregard me, just tried a LSP rename on that bad boy and there would be a *lot* of updates, also, it is sometimes human, sometimes not